04_environment/design.tex

```diff
@@ -0,0 +1,289 @@
+The model is described in Equation~\eqref{eq:Population Density}.
+
+\begin{equation}
+\frac{\mathrm{d} P}{\mathrm{~d} t}=k(N-P)
+\label{eq:Population Density}
+\end{equation}
+
+\begin{itemize}
+    \item $P$ is the total population. 
+    \item $N$ is the maximum deer population (carrying capacity\footnote{The maximum population size of a biological species that can be sustained in a specific environment.}).
+    \item $k$ is the growth coefficient.
+    \item $t$ is time.
+\end{itemize}
+
+In Figure~\ref{fig:Deer population over time}, the simulation of the implemented logistic model can be observed, with a growth coefficient of $0.2$ and $N$ of $8$. It can be seen that there is an unpredictable pattern to population changes, which would be interesting to see affecting the tier system and in extend, the deer and fish foraging strategies of the agents.
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/images/Deer population over time.PNG}
+    \caption{Deer population over time (turns) under Logistic modelling with a growth coefficient of 0.2.}
+    \label{fig:Deer population over time}
+\end{figure}
+
+Furthermore, In our implementation, $k$ and $N$ are constants. 
+
+Modelling the population (or linking it to other environmental elements) allows us to make sure that the system’s dynamics are varying, making it harder for islands to settle onto a single foraging method because it always results in better returns. Pareto optimality is therefore harder to achieve and thus maximising social welfare (see Definition~\ref{def:Maximising Social Welfare}) is trickier, requiring agents to make more informed decisions, recognizing possible patterns. Of course, that would be an easier task for the islands, if they had knowledge on what is affecting the population (i.e. disasters), whereas a logistic model function might be more abstract in the eyes of our agents, requiring more processing. 
+
+\subsubsection{Method of Resource Allocation}
+
+Moreover, there is a parameter to allow either an equal split of returns after foraging regardless of the input, or a proportional split dependent on the amount of resources inputted by each island. Both of these implementations work in accordance to the previous specification. Such change allows us to observe the concept of free-riding further, looking at how greedy islands behave in the case of uniform resource returns.
+
+The distribution method is based on two scalars. The first scales the inputs to the tiers, allowing the agent to choose how much they wish the tiers to cost. After which, the number of fish or deer caught is chosen by the tiers which goes into their respective distributions. The output of these distributions are then multiplied by the output scale. This allows the user to adjust the return amount each island can obtain, the output scalar should be above the cost of living in order to ensure the islands have enough resources to survive.  
```

Comment:
if you say the first the reader also expects a second somewhere if you say it's based on two - was looking for it in the writing and saw you put it in with after which -> would be clearer with using the word second as well to finish the pattern you started